packages/generator-single-spa/src/react/templates/.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
   "extends": [
-    "react-important-stuff",
+    <%if (typescript) { %>"ts-react-important-stuff",<%} else { %>"react-important-stuff",<% } %>
```

Comment:
This changed since your last review. Before, we had a separate `.eslintrc` for typescript. Now there's just this templated line.